.gitignore

```diff
@@ -22,6 +22,12 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+.idea/
+.bin/
+.vscode
+.vscode/
+bin/
```

Comment:
Thanks for adding files that should be ignored. `bin/` is duplicated, you can remove it